org.kde.kclock.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "org.kde.kclock",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kclock",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=xdg-data/sounds",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.Solid.PowerManagement",
+        "--own-name=org.kde.kclockd"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+        {
+            "name": "kirigamiaddons",
+            "config-opts": [ "-DBUILD_TESTING=OFF" ],
+            "buildsystem": "cmake-ninja",
+            "sources": [ { "type": "git", "url": "https://invent.kde.org/libraries/kirigami-addons.git", "branch": "release/21.05", "commit": "4724edc523c855bba54c34366c272e41ad022869" } ]
+        },
+        {
+            "name": "kclock",
+            "config-opts": [ "-DBUILD_TESTING=OFF" ],
+            "buildsystem": "cmake-ninja",
+            "sources": [ { "type": "git", "url": "https://invent.kde.org/plasma-mobile/kclock.git", "branch": "master", "commit": "397e390d8337840d1ed9b2479eae439674100b84" } ]
```

Comment:
You don't have a tag to use?

Reply:
I had to push some metadata changes after the release, so it'll be resolved at the next release cycle (in one month).

Reply:
You can add a patch to the packaging for now. It's not much work and allows to already use the tag.